src/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -3,12 +3,15 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics.CodeAnalysis;
+using System.IO;
 
 namespace System.Net.Http
 {
     [SuppressMessage("Microsoft.Serialization", "CA2229")]
     public class HttpRequestException : Exception
     {
+        private bool _allowRetry;
```

Comment:
readonly?  Or just make the property:
```C#
internal bool AllowRetry { get; }
```
and assign to that, letting the compiler create the backing readonly field.

Reply:
changed to auto property